src/elfpy/agent.py

```diff
@@ -2,15 +2,19 @@
 Implements abstract classes that control agent behavior
 """
 
+from __future__ import annotations  # types will be strings by default in 3.11
```

Comment:
Perhaps it makes sense to make a todo for this? Not a big deal either way, but good to keep track of things like this.

Reply:
I think as a general rule of thumb any time we increment the minimum python version we should go through all __future__ imports and update them.

Reply:
Yep, that makes sense 👍